edx-platform/pearson-vue-theme/lms/static/js/instructor_dashboard/data_download.js

```diff
@@ -197,6 +198,31 @@
                     }
                 });
             });
+            this.$license_usage_csv_btn.click(function() {
+                var url = dataDownloadObj.$license_usage_csv_btn.data('endpoint');
+                var errorMessage = gettext('Error generating license usage report. Please try again.');
+                dataDownloadObj.clear_display();
+                return $.ajax({
+                    type: 'POST',
+                    dataType: 'json',
+                    url: url,
+                    error: function(error) {
```

Comment:
Isn't it necessary to call dataDownloadObj.clear_display(); here? @ivanvgh

Reply:
No, since it is already called before the ajax call in Li.204.